### ISSUE TITLE
md-input placeholder broken

### ISSUE THREAD

**Issue by wlingke** [angular/material#3154]
a) Is the placeholder supposed to create a label? There are times where I'd prefer there just to be a placeholder and not have a label.
b) If it is going to create a label, it will need to be aligned

See codepen below:
http://codepen.io/anon/pen/MwmJPJ

**Comment by aksakalli** [comment]
It is possible by specifying it with `md-no-float`.

```
<md-input-container  md-no-float>
  <input placeholder="test" ng-model="data">
</md-input-container>
```

**Comment by wlingke** [comment]
Ah great, thanks for letting me know. b) remains unsolved but its a much less important issue.

**Comment by Splaktar** [comment]
Is this fixed in 0.10.0 or with #3582? What browser is this occurring in?

**Comment by wlingke** [comment]
This is good
